test/measureOperationsUtils.test.js

```diff
@@ -18,4 +22,113 @@ describe('measureOperationsUtils functions', () => {
     const REQUIRED_PARAMS = ['test', 'test2'];
     checkRequiredParams(req, REQUIRED_PARAMS, 'test');
   });
+
+  test('error thrown for unsupported $evaluate-measure params', async () => {
+    const UNSUPPORTEDREQ = {
+      query: { practitioner: 'testPractitioner', periodStart: '2019-01-01', periodEnd: '2019-12-31' }
+    };
+    try {
+      validateEvalMeasureParams(UNSUPPORTEDREQ);
+    } catch (e) {
+      expect(e.statusCode).toEqual(400);
+      expect(e.issue[0].details.text).toEqual(
+        `The following parameters were included and are not supported for $evaluate-measure: practitioner`
+      );
+    }
+  });
+
+  test('error thrown for unsupported $evaluate-measure reportType', async () => {
+    const UNSUPPORTEDREQ = {
+      query: { reportType: 'subject-list', periodStart: '2019-01-01', periodEnd: '2019-12-31' }
+    };
+    try {
+      validateEvalMeasureParams(UNSUPPORTEDREQ);
+    } catch (e) {
+      expect(e.statusCode).toEqual(501);
+      expect(e.issue[0].details.text).toEqual(`The subject-list reportType is not currently supported by the server.`);
+    }
+  });
+
+  test('error thrown for invalid $evaluate-measure reportType', async () => {
+    const INVALIDREQ = {
+      query: { reportType: 'invalid', periodStart: '2019-01-01', periodEnd: '2019-12-31' }
+    };
+    try {
+      validateEvalMeasureParams(INVALIDREQ);
+    } catch (e) {
+      expect(e.statusCode).toEqual(400);
+      expect(e.issue[0].details.text).toEqual(`reportType invalid is not supported for $evaluate-measure`);
+    }
+  });
+
+  test('error thrown for missing subject for $evaluate-measure', async () => {
+    const MISSING_SUBJECT_REQ = {
+      query: { reportType: 'individual', periodStart: '2019-01-01', periodEnd: '2019-12-31' }
+    };
+    try {
+      validateEvalMeasureParams(MISSING_SUBJECT_REQ);
+    } catch (e) {
+      expect(e.statusCode).toEqual(400);
+      expect(e.issue[0].details.text).toEqual(
+        `Must specify subject for all $evaluate-measure requests with reportType: individual`
+      );
+    }
+  });
+
```

Comment:
I would add a test here to check that validateCareGapsParams doesn't throw an error when passed valid parameters